pkgs/applications/audio/iconnconfig/default.nix

```diff
@@ -0,0 +1,47 @@
+{ lib
+, fetchgit
+, qt512
+, alsaLib
+, libusb1
+}:
+
+with qt512;
```

Comment:
please use the `libsForQt512.callPackage` function. And is there any specific reason why it needs to be that version? Also note, by using that callPackage function the attributes like `qtbase` are going to be in the lexical scope. So you will have to replace `qt512` with all the attributes in the scope of qt512 in the arguments list above.

Reply:
Replaced with libsForQt5 in all-packages